### PR TITLE
bpo-32651 Recommend getpass.getuser()

### PR DESCRIPTION
**Pull request by warsaw**
https://bugs.python.org/issue32651

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2018-01-24-10-42-57.bpo-32651.a_tZhY.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+In the :func:`os.getlogin()` documentation, recommend
+func:`getpass.getuser()`.
```
Comment (berkerpeksag) [inline Misc/NEWS.d/next/Documentation/2018-01-24-10-42-57.bpo-32651.a_tZhY.rst]: Missing ':' here. And use of parenthesis is not needed.

**Review by berkerpeksag — APPROVED** [review]
Since this is a small documentation improvement, a NEWS entry is not needed. If you still want to add it, I just left a comment :)

Do you plan to backport this to the 3.6 branch? I think it would be nice to get this into the 3.6 branch.

**Comment by warsaw** [comment]
@berkerpeksag Yeah, I guess that's true.  I've added the tag and will remove the file.

**Comment by warsaw** [comment]
I do plan on back porting this too, so I've added that label.

**Pull request merged**